app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -503,6 +507,8 @@ protected void setMediaPreviews(final List<Attachment> attachments, boolean sens
             setAttachmentClickListener(imageView, listener, i, attachment, true);
         }
 
+        mediaDescriptionIndicator.setVisibility(withDescription ? View.VISIBLE : View.GONE);
```

Comment:
This only shows a single "ALT", even if there are multiple images.

Although that is consistent with the sensitive media button.
And placing 4 of those stickers on the 4 preview images would be way too much clutter.

(Furthermore this is a bit "trusting" as one single character description is sufficient for it to show up, but I like that.)

Reply:
Twitter does it for each media and I think it makes sense?

Reply:
![0517e090ac3aed98](https://user-images.githubusercontent.com/1618905/204619368-5d85630e-7c1d-4661-aba1-5013ca4cf322.png)

Currently it looks like this here.

(It is also somewhat easier to understand in the layout code and code with only one item.)